ansible/roles/fews/files/glossis_workflow.yml

```diff
@@ -0,0 +1,121 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Workflow
+metadata:
+  generateName: glossis-
+spec:
+  entrypoint: complete-workflow
+  volumes:
+  - name: dgds-storage-cleanup
+    persistentVolumeClaim:
+      claimName: dgds2
+  - name: dgds-storage-waterlevel
+    persistentVolumeClaim:
+      claimName: dgds-waterlevel
+  - name: dgds-storage-currents
+    persistentVolumeClaim:
+      claimName: dgds-currents
+  - name: dgds-storage-wind
```

Comment:
Do we need this many volumes? Why can't there be 1 like gloffis?

Reply:
The volumes are RWO so if we want to run datasets in parallel we need a pvc for each dataset